pilot/cmd/pilot-agent/main.go

```diff
@@ -446,6 +453,8 @@ func getDNSDomain(podNamespace, domain string) string {
 }
 
 func init() {
+	proxyCmd.PersistentFlags().StringVar(&platform, "platform", "k8s",
```

Comment:
Please don't add this, it's hard to maintain and not consistent with the rest.

Reply:
We are moving away from flags. This needs to be mesh config or env var

Reply:
The platform flag is consumed by istio agent, not mesh wide setting, because a hybrid mesh can have workloads running on different platforms.